### ISSUE TITLE
cannot import name 'Figure' from 'bokeh.plotting'

### ISSUE THREAD

**Issue by ptal** [Dekker1/minizinc-slurm#29]
Hello,

I used the Python program provided in the README to visualize the results but I got the following error:

```
Traceback (most recent call last):
  File "/home/ptalbot/repositories/lattice-land/turbo/benchmarks/analysis.py", line 2, in <module>
    from mzn_bench.analysis.plot import plot_all_instances
  File "/home/ptalbot/repositories/mzn-bench/src/mzn_bench/analysis/plot.py", line 10, in <module>
    from bokeh.plotting import Figure, figure, gridplot
ImportError: cannot import name 'Figure' from 'bokeh.plotting' (/home/ptalbot/.local/lib/python3.10/site-packages/bokeh/plotting/__init__.py)
```

When I `pip list`, I do have bokeh 3.2.0 installed. I also have the same mzn-bench version than the one of this repository.

Any idea what's going on? 

Thanks!

**Comment by hbierlee** [comment]
Hi ptal,

Thanks for reporting the issue! When changing to poetry, we didn't fix the `bokeh` version, so you're running a newer version which has some API changes. I've resolved these changes and fixed the version in pyproject in #30, so once that's merged you should be able to run the plotting functionality (perhaps you need to update your depedencies by running `poetry install --all-extras`  (or `poetry install -E plotting`)).

Cheers,

Henk

Edit: small update, we are still going to use bokeh 3.1.1, so we can keep using python 3.8. In any case, installing with poetry should sort all of that out. Just be carefull to use the correct documentation version of Bokeh if you want to change the plots further.

**Comment by ptal** [comment]
Awesome thank you!
